api/libretimeapi/models/schedule.py

```diff
@@ -22,6 +22,20 @@ class Schedule(models.Model):
     def get_owner(self):
         return self.instance.get_owner()
 
+    def get_cueout(self):
+        if self.instance.ends < self.ends:
+            # Handle cases where the schedule item would overrun the show it is
+            # scheduled in
```

Comment:
What about using doc strings to explain the function globally ?

I am all for comment in the middle of the code, but this one could be rewritten as function doc string, and would allow code writers to access the doc string from their IDE.